resources/public/index.html

```diff
@@ -14,7 +14,7 @@
         <h1>Nano ID Collision Calculator</h1>
       </div>
       <div id="header-logo">
-        <img class="logo" src="logo.svg" alt="Nano ID logo by Anton Lovchikov" width="180" height="94">
+        <a href="https://github.com/zelark/nano-id-cc"><img class="logo" src="logo.svg" alt="Nano ID logo by Anton Lovchikov" width="180" height="94"></a>
```

Comment:
The link should point to the NanoID repository.

Reply:
Okay, will change it